[BUG] ABIEventFunctionNotFound

## Describe the bug

The following error occurred in our environment

```
[INDEXER-TransferApproval] [41] [ERROR] ("The event 'CancelTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
--
Traceback (most recent call last):
File "batch/indexer_transfer_approval.py", line 223, in __sync_token_cancel_transfer
events = ContractUtils.get_event_logs(
File "/app/ibet-Prime/batch/../app/utils/contract_utils.py", line 239, in get_event_logs
_event = getattr(contract.events, event)
File "/home/apl/.pyenv/versions/3.8.12/lib/python3.8/site-packages/web3/contract.py", line 253, in __getattr__
raise ABIEventFunctionNotFound(
web3.exceptions.ABIEventFunctionNotFound: ("The event 'CancelTransfer' was not found in this contract's abi. ", 'Are you sure you provided the correct contract abi?')
```

## Expected behavior

We have made the following fixes, but they were not sufficient.

https://github.com/BoostryJP/ibet-Prime/pull/281

## Versions to be fixed

- [x] v22.3
- [x] v22.6

--- a/app/utils/contract_utils.py
+++ b/app/utils/contract_utils.py
@@ -236,8 +236,8 @@ class ContractUtils:
         :param block_to: toBlock
         :return: Event logs
         """
-        _event = getattr(contract.events, event)
         try:
+            _event = getattr(contract.events, event)
             result = _event.getLogs(
                 fromBlock=block_from,
                 toBlock=block_to

